packages/test/src/test-integration-workflows.ts

```diff
@@ -216,6 +216,49 @@ test('Condition 0 patch sets a timer', async (t) => {
   t.false(await worker.runUntil(executeWorkflow(conditionTimeout0)));
 });
 
+export async function historySizeBytesGrows(): Promise<[number | undefined, number | undefined]> {
+  const before = workflow.workflowInfo().historySizeBytes;
+  await workflow.sleep(1);
+  const after = workflow.workflowInfo().historySizeBytes;
+  return [before, after];
+}
+
+test('HistorySizeBytes grows with new WFT', async (t) => {
+  const { createWorker, executeWorkflow } = helpers(t);
+  const worker = await createWorker();
+  const [before, after] = await worker.runUntil(executeWorkflow(historySizeBytesGrows));
+  t.assert(after && before && after > before && before > 100);
```

Comment:
Consistency: use `t.true()` instead of `t.assert()`. They are really doing the same anyway.

Reply:
done